app/controllers/dashboard_controller.rb

```diff
@@ -69,6 +69,7 @@ def app_config
       FACEBOOK_API_VERSION: GlobalConfigService.load('FACEBOOK_API_VERSION', 'v17.0'),
       WHATSAPP_APP_ID: GlobalConfigService.load('WHATSAPP_APP_ID', ''),
       WHATSAPP_CONFIGURATION_ID: GlobalConfigService.load('WHATSAPP_CONFIGURATION_ID', ''),
+      WHATSAPP_FEATURE_TYPE: GlobalConfigService.load('WHATSAPP_FEATURE_TYPE', ''),
```

Comment:
@hahuma  let remove this installation config, this seems like an overkill. Lets have this enabled by default if someone opts for embedded sign up